### ISSUE TITLE
Error occurs if there is not `migrations` table

### ISSUE THREAD

**Issue by ssi-anik** [sdispater/orator#214]
![screen shot 2018-01-22 at 8 10 33 pm](https://user-images.githubusercontent.com/2676602/35224781-5c7493e8-ffb0-11e7-994c-7ed799dc7d94.png)

**Comment by jychen7** [comment]
@ssi-anik 
`orator migrate` will auto create the `migrations` table, is it because you run `orator migrate:rollback` before `orator migarate`? looks like not a typical scenario.

**Comment by ssi-anik** [comment]
Yes. So, shouldn't we check if there is any migrations table or not?

**Comment by jychen7** [comment]
true, it should check
